app/workers/work-failer.ts

```diff
@@ -51,7 +51,7 @@ export default class WorkFailer implements Worker {
     const jobServiceThresholds = {};
 
     const workItems = await getWorkItemsByUpdateAgeAndStatus(
-      db, lastUpdateOlderThanMinutes, [WorkItemStatus.RUNNING],
+      db, lastUpdateOlderThanMinutes, [WorkItemStatus.RUNNING, WorkItemStatus.QUEUED],
```

Comment:
The work failer will now fail (cause a retry) for work items that have been in the 'queued' state for a long time, just like we do with the 'running' state.